src/libs/nav.js

```diff
@@ -51,13 +51,14 @@ export const Redirector = ({ pathname, search }) => {
   return null
 }
 
-const parseRoute = (handlers, { pathname, search }) => {
+const parseRoute = (handlers, { pathname, search, state }) => {
```

Comment:
So, is this unused? Should we revert this file?

Reply:
This is unused, we can revert it if that's the standard for this repo!